build.sbt

```diff
@@ -74,7 +79,9 @@
     "com.typesafe" % "config" % "1.2.1",
     slf4jApi,
     log4jOverSlf4j,
-    logbackClassic
+    logbackClassic,
+    bouncycastle,
```

Comment:
I'd use `baseJavaDependencies` just for java libraries that could also be used in java-only modules. The `crypto` dependencies can be added where we need them, e.g. in the domain module we currently have:
````scala
lazy val domain = Project(
     "domain",
     file("domain"),
     settings = scalaProjectSettings ++ Seq(
       libraryDependencies ++= testing
     )
   ).dependsOn(common % fullScope)
````
Which could be:
````scala
lazy val domain = Project(
     "domain",
     file("domain"),
     settings = scalaProjectSettings ++ Seq(
       libraryDependencies ++= testing ++ crypto
     )
   ).dependsOn(common % fullScope)
````